### PR TITLE
 Fix [#754]: Add config directory option for exim mailservers 

### PR DESCRIPTION
**Pull request by tsadpbb**

### CODE REVIEW

**Comment by sawolf** [comment]
LGTM!

File: NEWS [hunk 1]
```diff
@@ -1,4 +1,7 @@
 This file documents the major additions and syntax changes between releases.
+2.4.10 2024-XX-XX
```
Comment (sawolf) [inline NEWS:2]:
```suggestion
2.4.10 2024-04-30
```

**Comment by tsadpbb** [comment]
There is more at play here
https://www.exim.org/exim-html-current/doc/html/spec_html/ch-how_exim_receives_and_delivers_mail.html
https://www.exim.org/exim-html-4.61/doc/html/spec_html/ch-how_exim_receives_and_delivers_mail.html

Exim changed the structure of their message ID's at some point so the pattern matching on like 510 in check_mailq.pl no longer matches with the new message ID's.

**Comment by sawolf** [comment]
I continue to approve!

**Pull request merged**